kernelci/test.py

```diff
@@ -104,6 +105,7 @@ def get_params(bmeta, target, plan_config, storage):
 
     params = {
         'name': job_name,
+        'filename': job_filename,
```

Comment:
Ah, except the file name isn't strictly an attribute of the job.  I think it's OK but slightly on the edge of what should really be in this dictionary.  The job file name used when generating LAVA job definition files is a LAVA-specific thing, whereas `kernelci.test` is only supposed to produce lab-independent data.  Also the 200-character limit is a LAVA thing.

So I think it would be better to leave this part as-is in `kernelci.test` and only truncate the job name in `kernelci.lab.lava` to ensure it's under 200 characters so it doesn't fail to get submitted.  It's OK to edit the dictionary returned from `get_params()` so the job name can be truncated there directly.